boards/FSM/Src/App/App_AcceleratorPedals.c

```diff
@@ -221,7 +222,7 @@ void App_AcceleratorPedals_Broadcast(const struct FsmWorld *world)
         accelerator_pedals->app_brake_signal,
         App_Brake_IsBrakeActuated(brake) && accelerator_pedals->get_primary_pedal_percent() > 25,
         accelerator_pedals->get_primary_pedal_percent() < 5);
-    const bool brake_acc_disagreement = app_brake_disagreement == SIGNAL_STATE_ACTIVE;
+    const bool brake_acc_disagreement = false; // app_brake_disagreement == SIGNAL_STATE_ACTIVE;
     App_CanAlerts_SetFault(FSM_FAULT_BRAKE_ACC_DISAGREEMENT, brake_acc_disagreement);
```

Comment:
Fault removed: Papps/brake plausability no longer checked (since brake is permanently actuated due to HW bug)

Reply:
NOTE: SINCE BRAKE IS PERMANENTLY ACTUATED, BE CAREFUL ABOUT ENABLING DRIVE STATE. YOU DO NOT NEED TO BE PRESSING THE BRAKE TO ENTER DRIVE STATE. IF YOU ARE PRESSING DOWN ACCELERATOR PEDAL AND FLIP THE START SWITCH, THE CAR WILL DRIVE.

Reply:
uhhh I'm assuming we don't want to leave this in?

Reply:
yep reverted this file